src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -42,8 +43,12 @@ export default function FacilityAddress({
       {isHomepageRefresh &&
         !!phone && (
           <div className="vads-u-margin-top--1p5">
-            <strong>Clinic: </strong> {clinicFriendlyName}
-            <br />
+            {!isCommunityCare && (
```

Comment:
Instead of adding an `isCommunityCare` prop, you could replace this check with `!!clinicFriendlyName`. The CC details page won't ever pass a clinic name to this component.